
public class exception {
    public static void main(String[] args) {
    try{ int a[]={2,6,4,7};
      System.out.println(a[5]);
    }

 catch (Exception e) {
System.out.println(e);
 }
 finally {
  System.out.println("hello");
}

}
}
